eslint.config.mjs

```diff
@@ -0,0 +1,207 @@
+import eslintPluginVue from 'eslint-plugin-vue'
+import vuejsAccessibility from 'eslint-plugin-vuejs-accessibility'
+import eslintPluginUnicorn from 'eslint-plugin-unicorn'
+import eslintConfigPrettier from 'eslint-config-prettier'
+import intlifyVueI18N from '@intlify/eslint-plugin-vue-i18n'
+import globals from 'globals'
+import vueEslintParser from 'vue-eslint-parser'
+import path from 'node:path'
+import { fileURLToPath } from 'node:url'
+import js from '@eslint/js'
+import { FlatCompat } from '@eslint/eslintrc'
+import { fixupConfigRules } from '@eslint/compat'
+import jsoncEslintParser from 'jsonc-eslint-parser'
+import eslintPluginJsonc from 'eslint-plugin-jsonc'
+import eslintPluginYml from 'eslint-plugin-yml'
+import yamlEslintParser from 'yaml-eslint-parser'
+
+const { default: activeLocales } =
+  await import('./static/locales/activeLocales.json', { with: { type: 'json' } })
+
+const __filename = fileURLToPath(import.meta.url)
+const __dirname = path.dirname(__filename)
+const compat = new FlatCompat({
+  baseDirectory: __dirname,
```

Comment:
We can use `import.meta.dirname` here, as it is supported in Node 21.2.0+ and LTS 20.11.0+ and we already use it in the `_scripts/patchShaka.mjs` file.

```suggestion
const compat = new FlatCompat({
  baseDirectory: import.meta.dirname,
```